### PR TITLE
typography extend

### PR DESCRIPTION
**Pull request by grandleemon**

### CODE REVIEW

File: docusaurus/docs-internals/editor-controls/multivalue-controls/typography.md [hunk 1]
```diff
@@ -326,3 +379,105 @@ Determines whether the text can be formatted as subscript or superscript.
...
+  type: "typography",
+  default: {
+    fontSize: 16,
+    fontSizeSuffix: "px",
+    bold: true,
+    italic: true,
+    fontFamily: "lato"
+  }
+}
+```
```
Comment (grandleemon) [inline docusaurus/docs-internals/editor-controls/multivalue-controls/typography.md:399]: doesn't work in edtiro

File: docusaurus/docs-internals/editor-controls/multivalue-controls/typography.md [hunk 2]
```diff
@@ -326,3 +379,124 @@ Determines whether the text can be formatted as subscript or superscript.
...
+  style: ({ value }) => {
+    return {
+      "{{WRAPPER}} .brz-text": {
+        "font-weight": value.fontWeight,
+        "letter-spacing": value.letterSpacing
+      }
+    }
+  }
+}
+```
```
Comment (grandleemon) [inline docusaurus/docs-internals/editor-controls/multivalue-controls/typography.md:418]: `style` doesn't trigger in editor

**Review by victor-pg — CHANGES_REQUESTED** [review]
Fix the error because doc are not running

[Screencast from 06-21-2024 01:47:34 PM.webm](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/e184b18c-dd2a-4e0b-bcee-0c88c1a6d3a6)

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image
2. `style` is triggered:

[Screencast from 06-25-2024 11:16:45 AM.webm](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/b3dc06ab-fbb3-4665-bb61-65837e23021c)

3. image
4. Missing `selector` in parameters + example
5. image
6. image

**Pull request merged**